frame/evm/src/weights.rs

```diff
@@ -0,0 +1,53 @@
+
+//! Autogenerated weights for `pallet_evm`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-03-03, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/frontier-template-node
+// benchmark
+// --chain
+// dev
+// --execution
+// wasm
+// --wasm-execution
+// compiled
+// --pallet
+// pallet_evm
+// --extrinsic
+// hotfix_inc_account_sufficients
+// --steps
+// 20
+// --repeat
+// 10
+// --json-file=raw.json
+// --output
+// frame/evm/src/weights.rs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for `pallet_evm`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> pallet_evm::WeightInfo for WeightInfo<T> {
+	// Storage: System Account (r:200 w:200)
+	// Storage: System Number (r:1 w:0)
+	// Storage: System ExecutionPhase (r:1 w:0)
+	// Storage: System EventCount (r:1 w:1)
+	// Storage: System Events (r:1 w:1)
+	fn hotfix_inc_account_sufficients(n: u32, ) -> Weight {
+		(436_991_000 as Weight)
+			// Standard Error: 56_000
+			.saturating_add((9_964_000 as Weight).saturating_mul(n as Weight))
+			.saturating_add(T::DbWeight::get().reads(4 as Weight))
+			.saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(n as Weight)))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(n as Weight)))
```

Comment:
You know this already, just leaving it here for the record: this needs to be done on official benchmarking hardware